scli/main.py

```diff
@@ -84,8 +85,27 @@ def destroy_ssh_tunnels():
 @click.option('--dc', help='Datacenter to repair')
 @click.option('--local', is_flag=True, help='Repair using hosts in local DC '
                                             'only')
-def repair(keyspace, table, hosts, exclude, dc, local):
+@click.option('-h', '--host', envvar='SCYLLA_HOST')
+@click.option('-u', '--ssh_username', envvar='SCYLLA_USERNAME', default='',
+              help='SSH username on Scylla host')
+@click.option('-k', '--ssh_pkey', envvar='SCYLLA_PKEY',
+              default=os.path.expanduser('~/.ssh/id_rsa'),
+              help='SSH public key path')
+@click.option('-p', '--ssh_pass', is_flag=True,
+              help='Use this flag if your SSH key is protected by password')
+@click.option('-l', '--log_to', help='Where to store logs from the client')
+@click.pass_context
+def repair(ctx, host, keyspace, table, hosts, exclude, dc, local, ssh_username, ssh_pkey, ssh_pass, log_to):
+    if host is None:
+        click.echo('Either --host or SCYLLA_HOST env should be provided')
+        raise click.Abort()
+
+    if ssh_username != "":
```

Comment:
I don't like this part (initializing ssh tunnel) being repeated this way for every command.

Reply:
Ok @r4fek  say me if you prefer with the commit `6d23a5a`

Reply:
Sorry again for the delay. Looks good!